### DISCUSSION TITLE
How to add some example NTFs in the "owned NFTs"

### DISCUSSION THREAD

**Discussion by PhantomWhite233** [ConsenSys-archive/truffle#6003] [General Q&A]
In the course Build a NFT rental marketplace Part 2, I successfully built my own NFT rental marketplace according to the video, but I saw that some examples of NFT could be added to the "owned NFTs" in the video. I wonder how this is done

**Comment by FlaviuWeb3** [comment]
@PhantomWhite233 could you please elaborate a bit more about your request?
